posh-git.psm1

```diff
@@ -11,26 +11,79 @@ if ($psv.Major -lt 3 -and !$NoVersionWarn) {
     "To suppress this warning, change your profile to include 'Import-Module posh-git -Args `$true'.")
 }
 
-Push-Location $psScriptRoot

```

Comment:
There was a pop-location later but if something crashed out before the pop-location, it would have left the current location to the module's root dir.